woocommerce/woocommerce-ios
# [CPP Onboarding] Implement Stripe account restricted - pending requirements

Part of #4611 

This adds an specific error when the Stripe account is restricted because there are pending requirements. 

<img src="https://user-images.githubusercontent.com/2722505/128891854-ab530e33-73d0-4a72-a34d-588c4f961ceb.png" width="350"/>

As with similar screens, the contact us and learn more buttons should be actionable:

| Contact us | Learn more |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/128892143-73f5ec8a-fe6e-4ee3-8b1e-0953aed64692.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/128892113-574f38d4-6008-4e66-a978-1d8b27140a03.png" width="350"/> |


## Changes
* Add the new view to render the error screen
* The deadline for those requirements is propagated to ui

## How to test
In order to test this in real conditions, it is necessary to have a store associated to an Stripe account that is restricted because there are pending requirements or...
make `CardPresentPaymentsOnboardingUseCase.checkOnboardingState` return `stripeAccountPendingRequirement`

1. Navigate to Settings > In-Person Payments
2. Notice the error screen


Update release notes:
